includes/REST_API/Stories_Controller.php

```diff
@@ -157,6 +183,13 @@ public function get_item_schema() {
 			'default'     => '',
 		];
 
+		$schema['properties']['color_presets'] = [
+			'description' => __( 'Color presets used by all stories', 'web-stories' ),
+			'type'        => 'array',
```

Comment:
Array requires `'items'` property here, otherwise you get a PHP warning.

Reply:
An example of items in use - https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php#L441-L445

Reply:
Thanks! Added `object` as items.